packages/issuer-api/src/pods/certificate/certificate.controller.ts

```diff
@@ -30,13 +30,16 @@ import { CertificateEvent } from '../../types';
 import { GetAllCertificateEventsQuery } from './queries/get-all-certificate-events.query';
 import { CertificateDTO } from './certificate.dto';
 import { SuccessResponseDTO } from '../../utils/success-response.dto';
+import { ExceptionController } from '../../utils/ExceptionController';
 
 @ApiTags('certificates')
 @Controller('certificate')
-export class CertificateController {
+export class CertificateController extends ExceptionController {
```

Comment:
I guess ExceptionController  should be just utils function, would be more re-usable etc

Reply:
Alternatively, you could implement a interceptor to transform return into errors if necessary for e.g like https://github.com/energywebfoundation/origin/blob/d1025d2b034f317520de03866ff4b76bb62c333a/packages/origin-backend-utils/src/interceptor/NullOrUndefinedResultInterceptor.ts

Reply:
Good idea, interceptor is a better solution.